SlidingSegmentedControlTests/SlidingSegmentedControlTests.swift

```diff
@@ -507,3 +639,37 @@ struct SegmentLocatorStub: SegmentLocator {
     }
 
 }
+
+class AnimatingMock: Animating {
+
+    struct EquatableParams: Equatable {
+        let duration: TimeInterval
+        let delay: TimeInterval
+        let dampingRatio: CGFloat
+        let velocity: CGFloat
+        let options: UIViewAnimationOptions
+    }
+
+    static var capturedEquatableParamsArray: [EquatableParams] = []
```

Comment:
Consider removing unnecessary type repetition (`Array`) and improving non-descriptive name (`equatableParams`).